pkg/processor/trigger/kafka/test/kafka_test.go

```diff
@@ -717,6 +738,52 @@ func (suite *testSuite) resolveReceivedEventBodies(deployResult *platform.Create
 	return receivedBodies
 }
 
+func (suite *testSuite) ensureNumberOfCommittedOffsets(consumerGroup string, topic string, expectedNumberOfCommittedOffsets int) bool {
```

Comment:
Ensure usually means that if some condition doesn't happen, the function makes sure it does.

```suggestion
func (suite *testSuite) validateNumberOfCommittedOffsets(consumerGroup, topic string, expectedNumberOfCommittedOffsets int) bool {
```